### DISCUSSION TITLE
Is there any way to test regress test in risedev tools?

### DISCUSSION THREAD

**Discussion by thexiay** [risingwavelabs/risingwave#13254] [Ideas]
Is there any way to test regress test in risedev tools?  
For example, if i issue a pull request，i found my regress test fail, is there a quick and easy way to verify it locally？

**Comment by wangrunji0408** [comment]
Here is the command I used:
```sh
# run risingwave
./risedev p

# run regress test
rm -rf `pwd`/src/tests/regress/output
RUST_BACKTRACE=1 cargo r -p risingwave_regress_test -- --host 127.0.0.1 \
  -p 4566 \
  -u root \
  --input `pwd`/src/tests/regress/data \
  --output `pwd`/src/tests/regress/output \
  --schedule `pwd`/src/tests/regress/data/schedule \
  --mode risingwave
```
More details can be found in the [doc]( https://github.com/risingwavelabs/risingwave/blob/13d29575b0107361049a5bb7c02175aad42bacd9/src/tests/regress/README.md#L4).

**Comment by wangrunji0408** [comment]
Sorry. Just see your issue #13255 here. Yes, we should provide an easier way to run regress test.

**Comment by thexiay** [comment]
> Sorry. Just see your issue #13255 here. Yes, we should provide an easier way to run regress test.

thanks for your reply🤩I would like to add this small tool into risedev
